bpf/cpu/cpu.bpf.c

```diff
@@ -1,47 +1,43 @@
 // +build ignore
```

Comment:
We should add a license text here. Something like:

// SPDX-License-Identifier: GPL-2.0-only
/* Copyright 2022 The Parca Authors */